ci-operator/step-registry/ovn/conf/dualstack-manifest/ovn-conf-dualstack-manifest-commands.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cat >> "${SHARED_DIR}/manifest_cluster-network-03-config.yml" << EOF
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: IPv6DualStackNoUpgrade
```

Comment:
* Cluster operator kube-apiserver cannot be upgraded between minor versions: FeatureGates_RestrictedFeatureGates_IPv6DualStackNoUpgrade: FeatureGatesUpgradeable: "IPv6DualStackNoUpgrade" does not allow updates